src/app/retrospective/sprint-list/sprint-list.component.ts

```diff
@@ -38,15 +51,26 @@ export class SprintListComponent implements OnInit {
 
     refresh() {
         this.initializeDataSource();
-        this.changeDetectorRefs.detectChanges();
     }
 
     initializeDataSource() {
+        const documentBody = document.body, documentElement = document.documentElement;
+        // Taking the maximum value for the height of the page as it is
+        // calculated differently by different Browsers.
+        const pageHeight = Math.max( documentBody.scrollHeight, documentBody.offsetHeight,
+            documentElement.clientHeight, documentElement.scrollHeight, documentElement.offsetHeight );
+
+        // Passing height/48 as rowsToLoad as min-height of
+        // mat-row is set to 48 by default.
```

Comment:
Add some reference links

Reply:
You can see [here](https://github.com/angular/material2/blob/2046b7a3287bbff87a48800cf7d770f1976ed757/src/lib/table/table.scss#L2) that $mat-row-height is set to 48px.

You can see [here](https://github.com/angular/material2/blob/2046b7a3287bbff87a48800cf7d770f1976ed757/src/lib/table/table.scss#L17) that min-height of mat-row is set to `$mat-row-height`.